migration/iD10/postgresql/202303081200_IDEMPIERE-5611.sql

```diff
@@ -0,0 +1,297 @@
+;
+
+UPDATE AD_Table SET AD_Window_ID=53038 /*Payroll Definition*/ WHERE TableName='HR_PayrollConcept'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53038 /*Payroll Definition*/ WHERE TableName='HR_Year'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53028 /*Import Processor*/ WHERE TableName='IMP_ProcessorLog'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53028 /*Import Processor*/ WHERE TableName='IMP_ProcessorParameter'
+;
+
+UPDATE AD_Table SET AD_Window_ID=354 /*Product BOM*/ WHERE TableName='M_BOMAlternative'
+;
+
+UPDATE AD_Table SET AD_Window_ID=344 /*Product Costs*/ WHERE TableName='M_CostMovement_v'
+;
+
+UPDATE AD_Table SET AD_Window_ID=358 /*Attribute Set Instance*/ WHERE TableName='M_InOutLineMA_v'
+;
+
+UPDATE AD_Table SET AD_Window_ID=358 /*Attribute Set Instance*/ WHERE TableName='M_MovementLineMA_v'
+;
+
+UPDATE AD_Table SET AD_Window_ID=200025 /*Shipment Package*/ WHERE TableName='M_PackageMPS'
+;
+
+UPDATE AD_Table SET AD_Window_ID=140 /*Product*/ WHERE TableName='M_ProductDownload'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53149 /*Quality Test*/ WHERE TableName='M_Product_QualityTest'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53074 /*Promotion*/ WHERE TableName='M_Promotion'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53074 /*Promotion*/ WHERE TableName='M_PromotionDistribution'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53073 /*Promotion Group*/ WHERE TableName='M_PromotionGroup'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53073 /*Promotion Group*/ WHERE TableName='M_PromotionGroupLine'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53074 /*Promotion*/ WHERE TableName='M_PromotionLine'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53074 /*Promotion*/ WHERE TableName='M_PromotionPreCondition'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53074 /*Promotion*/ WHERE TableName='M_PromotionReward'
+;
+
+UPDATE AD_Table SET AD_Window_ID=140 /*Product*/ WHERE TableName='M_StorageReservationLog'
+;
+
+UPDATE AD_Table SET AD_Window_ID=50007 /*Dashboard Content*/ WHERE TableName='PA_DashboardContent_Access'
+;
+
+UPDATE AD_Table SET AD_Window_ID=200006 /*Dashboard Preference*/ WHERE TableName='PA_DashboardPreference'
+;
+
+UPDATE AD_Table SET AD_Window_ID=200090 /*Document Status*/ WHERE TableName='PA_DocumentStatusAccess'
+;
+
+UPDATE AD_Table SET AD_Window_ID=358 /*Attribute Set Instance*/ WHERE TableName='RV_C_InvoiceLine'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53165 /*Customer*/ WHERE TableName='RV_OrderDetail'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53064 /*View MRP Records*/ WHERE TableName='RV_PP_MRP'
+;
+
+UPDATE AD_Table SET AD_Window_ID=232 /*Request (all)*/ WHERE TableName='RV_RequestUpdates'
+;
+
+UPDATE AD_Table SET AD_Window_ID=52003 /*Web POS BlackListCheque*/ WHERE TableName='U_BlackListCheque'
+;
+
+UPDATE AD_Table SET AD_Window_ID=53065 /*Web POS Terminal*/ WHERE TableName='U_POSTerminal'
+;
+
+UPDATE AD_Table SET AD_Window_ID=52001 /*Web POS Role Menu*/ WHERE TableName='U_RoleMenu'
+;
+
+UPDATE AD_Table SET AD_Window_ID=52000 /*Web POS Menu*/ WHERE TableName='U_WebMenu'
+;
+
```

Comment:
hi @CarlosRuiz-globalqss , is there a risk here that this will update customization of existing implementations ?